include/mysql.h

```diff
@@ -323,6 +331,7 @@ struct st_mysql_options {
     char *bind_address;
     my_bool secure_auth;
     my_bool report_data_truncation; 
+    enable_redirect redirection_mode;
```

Comment:
New members of mysql_options should be placed in mysql_options->extension (For example check MYSQL_PROGESS_CALLBACK)